compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -529,6 +529,12 @@ def PadDynamicAllocPass :
 def PolynomialApproximationPass :
     Pass<"iree-codegen-polynomial-approximation", ""> {
   let summary = "Convert math operations to their polynomial approximation";
+  let options = [
+    ListOption<"noApproxOps", "no-approx-ops", "std::string",
```

Comment:
drive by: maybe rename it as `denyList`?

Reply:
I like the current name more, it's more specific to what the pass is doing

Reply:
I actually think we should just reverse it -- list the ops you want to do approx in the list.